djangosaml2/acs_failures.py

```diff
@@ -5,18 +5,9 @@
 #
 from __future__ import unicode_literals
 
-from django.core.exceptions import PermissionDenied
 from django.shortcuts import render
 
 
-def template_failure(request, status=403, **kwargs):
-    """ Renders a SAML-specific template with general authentication error description. """
-    return render(request, 'djangosaml2/login_error.html', status=status)
-
-
-def exception_failure(request, exc_class=PermissionDenied, **kwargs):
-    """ Rather than using a custom SAML specific template that is rendered on failure,
-    this makes use of a standard exception handling machinery present in Django
-    and thus ends up rendering a project-wide error page for Permission Denied exceptions.
-    """
-    raise exc_class
```

Comment:
This failure function was there as an easy drop-in replacement for the default template response. Usable if somebody wants to use the standard project-wide error handling mechanism instead.
Yeah, it would not hurt to re-raise the original exception instead; it is just an example anyway.

Reply:
If they are already looking to customize the error handler, than users are probably 'advanced' enough to know they can make a handler which just reraises the exception. This is something which could also be put in the readme as an example, rather than having this be unused in the code.

Reply:
Alright, it's a trivial thing, so whatever... :-)